test/ably/restchannelpublish_test.py

```diff
@@ -23,6 +23,7 @@
 log = logging.getLogger(__name__)
 
 
+@pytest.mark.filterwarnings('ignore::DeprecationWarning')
```

Comment:
I think we should add a comment above each of these deprecation warnings explaining that it's ignoring a deprecation warning emitted by the library itself.

Reply:
Okay, will do that for all occurences